src/main/java/net/openhft/chronicle/core/values/LongValue.java

```diff
@@ -75,4 +79,8 @@ default boolean isClosed() {
     @Override
     default void close() {
     }
+
+    default ThreadLock vanillaThreadLock() {
```

Comment:
The name is silent whether or not a new instance is created on each invocation. Perhaps "createThreadLock()" would be more descriptive?